src/SignUpIn/SignIn.js

```diff
@@ -1,16 +1,24 @@
 // No log in functionality yet
+
 import React, { Component } from 'react';
+import firebase from '../FirebaseConfig';
 
 class SignIn extends Component {
+
+    constructor(props) {
+        super(props);
+        this.nextStep = this.nextStep.bind(this);
+    }
+
     render() {
         return (
-            <form>
+            <form onSubmit={this.nextStep}>
                 <div className="login-wrapper">
                     <div className="login-input-wrapper">
-                        <input type="text" id="email" className="login-input form-component" placeholder="Email" /><br />
-                        <input type="password" id="password" className="login-input form-component" placeholder="Password" /><br />
+                        <input name="email" type="text" id="email" className="login-input form-component" placeholder="Email" /><br />
+                        <input name="password" type="password" id="password" className="login-input form-component" placeholder="Password" /><br />
                     </div>
-                    <div className="login-button-wrapper"><button type="button" className="login-button">Login</button></div>
+                    <div className="login-button-wrapper"><button type="submit" className="login-button">Login</button></div>
                     <div className="forgot">
                         <p className="forgot">forgot password?</p>
                         <p className="forgot">forgot username?</p>
```

Comment:
Add TODOs for these links to reset password/username

Reply:
Done